classes/core/AltapayTokenControl.php

```diff
@@ -105,19 +104,17 @@ public function creditCardCustomDescription( $description, $payment_id ) {
 
 				if ( $gateway->enabled === 'yes' && isset( $gateway->settings['token_control'] )
 					&& $gateway->settings['token_control'] === 'yes'
-				) {
-
-					if ( $gateway->id === $payment_id && is_user_logged_in() ) {
-						ob_start();
-						global $wpdb;
+					&& $gateway->id === $payment_id && is_user_logged_in() ) {
+					ob_start();
+					global $wpdb;
 
-						$userID  = get_current_user_id();
-						$results
-								= $wpdb->get_results( "SELECT * FROM {$wpdb->prefix}altapayCreditCardDetails WHERE userID='$userID'" );
+					$userID  = get_current_user_id();
+					$results = $wpdb->get_results( "SELECT * FROM {$wpdb->prefix}altapayCreditCardDetails WHERE userID='$userID'" );
 
+					if ( $results ) {
 						$creditCard[] = _( 'Select a saved credit card' );
 						foreach ( $results as $result ) {
-							$creditCard[ $result->creditCardNumber ] = $result->creditCardNumber; // masked credit card number
+							$creditCard[ $result->creditCardNumber ] = '**********' . $result->creditCardNumber; // Last four digits
```

Comment:
There is a `lastFourDigits` in the response.

Reply:
@emicha That is the name of the column in the plugin database table and we are storing the last four digits in this column from the response. `['CardInformation']['LastFourDigits']`
<img width="608" alt="Screen Shot 2022-07-28 at 9 38 37 AM" src="https://user-images.githubusercontent.com/39646587/181461958-9af4be15-739b-4260-8af7-fb6585fefe51.png">